src/client.rs

```diff
@@ -157,16 +168,51 @@ impl DeezerClient {
         Ok(res.data)
     }
 
-    async fn get<T: DeserializeOwned>(&self, url: &str) -> Result<T> {
-        let res = self
+    pub(crate) async fn get_subresource<T>(&self, id: u64, limit: Option<u32>,
+                                           offset: Option<u32>) -> Result<Vec<T>>
+        where
+            T: DeezerObject
+    {
+        let url = T::get_api_url(id);
+        let url = format!("{}/{}", BASE_URL, url);
+
+        let mut params: HashMap<String, String> = HashMap::new();
+        if let Some(_limit) = limit {
```

Comment:
You can actually just use `limit` and `offset` as variable names. The underscore indicated an unused variable so it shouldn't be used here.

Reply:
changed